Dockerfile

```diff
@@ -29,3 +29,12 @@ RUN sdkmanager "platform-tools"
 RUN sdkmanager $(sdkmanager --list 2> /dev/null | grep platforms | awk -F' ' '{print $1}' | sort -nr -k2 -t- | head -8)
 # list all build-tools, sort them in descending order and install them
 RUN sdkmanager $(sdkmanager --list 2> /dev/null | grep build-tools | awk -F' ' '{print $1}' | sort -nr -k2 -t \; | uniq)
+
+# emulator
+
+# install packages
+RUN sdkmanager "emulator"
```

Comment:
I would run all emulator-related commands within one `RUN` instruction, so that it creates only one layer in the Docker image.